cfroutesync/integration/testenv_test.go

```diff
@@ -19,10 +19,13 @@ import (
 
 	"k8s.io/client-go/rest"
 
+	"github.com/onsi/gomega/gexec"
+
 	"code.cloudfoundry.org/cf-k8s-networking/cfroutesync/ccclient"
 	"code.cloudfoundry.org/cf-k8s-networking/cfroutesync/cfg"
 	log "github.com/sirupsen/logrus"
 	fakeapiserver "sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
```

Comment:
Curious about this, do you think we should be using this same logger in our main code?

Reply:
this is just so that I can call `SetLogger` down below, so that our test output (when run with `-v`) includes the output of the `kube-apiserver`